stories/datepicker/datepicker.stories.js

```diff
@@ -0,0 +1,1684 @@
+                                <span class="fd-calendar__text" role="button">5</span>
+                            </td>
+                            <td role="gridcell" class="fd-calendar__item">
+                                <span class="fd-calendar__text" role="button">26</span></td>
+                            <td role="gridcell" class="fd-calendar__item">
+                                <span class="fd-calendar__text" role="button">27</span></td>
+                            <td role="gridcell" class="fd-calendar__item">
+                                <span class="fd-calendar__text" role="button">28</span></td>
+                            <td role="gridcell" class="fd-calendar__item">
+                                <span class="fd-calendar__text" role="button">29</span></td>
+                            <td role="gridcell" class="fd-calendar__item">
+                                <span class="fd-calendar__text" role="button">30</span></td>
+                            <td role="gridcell" class="fd-calendar__item fd-calendar__item--other-month">
+                                <span class="fd-calendar__text" role="button">1</span></td>
+                            <td role="gridcell" class="fd-calendar__item fd-calendar__item--other-month">
+                                <span class="fd-calendar__text" role="button">2</span></td>
+                        </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+        <footer class="fd-dialog__footer fd-bar fd-bar--footer fd-bar--cosy">
+            <div class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button class="fd-dialog__decisive-button fd-button fd-button--emphasized">
+                        OK
+                    </button>
+                </div>
+            </div>
+        </footer>
+    </div>
+</div>
+<br />
+<div class="fd-dialog-docs-static fd-calendar-mobile-docs-static--portrait fd-dialog fd-dialog--active" dir="rtl">
+    <div class="fd-dialog__content fd-dialog__content--s fd-dialog__content--mobile">
+        <header class="fd-dialog__header fd-bar fd-bar--header fd-bar--cosy">
+            <div class="fd-bar__left">
+                <div class="fd-bar__element">
+                    <h3 class="fd-dialog__title">
+                        Pick a date
+                    </h3>
+                </div>
+            </div>
+            <div class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button type="button" class="fd-button fd-button--transparent fd-calendar__close-button" aria-label="Close"></button>
+                </div>
+            </div>
+        </header>
+        <div class="fd-dialog__body fd-dialog__body--no-vertical-padding">
+            <div class="fd-calendar fd-calendar--mobile-portrait">
+                <header class="fd-calendar__header">
```

Comment:
can this be changed to
```html
<div class="fd-calendar__header">
```
There should not be 2 headers on the page. Also since the application page might already have a [header that houses the app logo and navigational content](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) - I think all the `<header>` usages in Calendar should be changed to `<div>`.